bonsai-core/src/test/scala/com/stripe/bonsai/BitsetSpec.scala

```diff
@@ -18,7 +18,7 @@ class IndexedBitSetSpec extends WordSpec with Matchers with Checkers {
 
   implicit val arbitraryBitSetWithIndex: Arbitrary[BitSetWithIndex] =
     Arbitrary(for {
-      len <- arbitrary[Short].map(_.toInt)
+      len <- arbitrary[Short].map(_.toInt) if len >= 0
```

Comment:
Can we instead change this to something like the following?

```scala
arbitrary[Short].map(_ & 0xffff)
```

This way we get a non-negative value without any filtering (which I think is best to avoid when possible).